integration/docker_popular_images/README

```diff
@@ -2,6 +2,6 @@
 
 To run these tests you just do
 
-bats popular-containers-images.bats
+bats popular_docker_images.bats
```

Comment:
obviously correct :-)
but, whilst here, as I checked - I see we have the `popular_docker_images.bats` file living under the `docker_popular_images` folder. How would you feel about unifying them to be the same word order whilst fixing this doc? :-)